addons/mail/static/tests/message/message_reply.test.js

```diff
@@ -120,3 +121,28 @@ test("click on message in reply highlights original message", async () => {
     );
     await contains(".o-mail-Message.o-highlighted:contains('This message has been removed')");
 });
+
+test("reply action functionality in Chatter", async () => {
+    const pyEnv = await startServer();
+    const partnerBId = pyEnv["res.partner"].create({ name: "Partner B" });
+    pyEnv["mail.message"].create({
+        body: "Test message from B",
+        subtype_id: pyEnv["mail.message.subtype"].search([
+            ["subtype_xmlid", "=", "mail.mt_note"],
+        ])[0],
+        model: "res.partner",
+        res_id: serverState.partnerId,
+        author_id: partnerBId,
+    });
+    await start();
+    await openFormView("res.partner", serverState.partnerId);
+    await click(".o-mail-Message-actions button[title='Reply']");
+    await contains("button.active", { text: "Log note" });
+    await contains(".o-mail-Composer-coreHeader", { text: "Replying to Partner B", count: 0 });
```

Comment:
Negative assertion with a matching positive selector has very poor value in test.
Reason is that the test could fail functionally but will still pass because this assertion doesn't match the actual selector, i.e. we renamed the structural classnames or the string is slightly changed.

I don't think this part is that useful for the test. If there's a bug in the future about it maybe best to have it fixed with appropriate test than be too greedy with test coverage. Reminder that each test is more code maintenance so there's a balance of adding test coverage for the right amount needed.